e2e/e2e_test.go

```diff
@@ -26,17 +27,25 @@ import (
 
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/api/whereabouts.cni.cncf.io/v1alpha1"
 	wbclient "github.com/k8snetworkplumbingwg/whereabouts/pkg/client/clientset/versioned"
+	"github.com/k8snetworkplumbingwg/whereabouts/pkg/logging"
+	"github.com/k8snetworkplumbingwg/whereabouts/pkg/reconciler"
 	wbstorage "github.com/k8snetworkplumbingwg/whereabouts/pkg/storage/kubernetes"
+	"github.com/k8snetworkplumbingwg/whereabouts/pkg/types"
 )
 
 const (
-	testNetworkName = "wa-nad"
-	testNamespace   = "default"
-	testImage       = "quay.io/dougbtv/alpine:latest"
-	ipv4TestRange   = "10.10.0.0/16"
-	singlePodName   = "whereabouts-basic-test"
-	createTimeout   = 10 * time.Second
-	deleteTimeout   = 2 * createTimeout
+	wbLabelEqual     = "tier=whereabouts-scale-test"
+	testNamespace    = "kube-system"
```

Comment:
This is why the stateful set pods stopped being ready: their pods are looking for the network-attachment-definition in the default namespace. 

I recommend you use that namespace as well.